app/index.ts

```diff
@@ -68,22 +68,24 @@ const unwrap = (testName: string, userFn: (dbData: { dbNameForThisTest: string;
 };
 
 // Initialize the DB
-void getMainDataSource().createTemplateDB();
+void getMainDataSource().createOrConnectToExistingTemplateDB();
```

Comment:
Why should we connect to an existing DB here?
This is the place to create a new template DB, no?